resource-service/README.md

```diff
@@ -18,46 +18,81 @@ limitations under the License.
 
 # Resource Service
 
-The Resource service accepts an incoming message from the `user` Kafka topic which contains the resourceId that is being accessed 
-(this could be an actual file, or a directory that could contain many files and/or sub-directories). The service will then query the backing store to
-see if the requested resourceId has been stored. If this is not the case then the request will be passed onto the local implementation of 
-the Resource service. All the returned resources will be within an Akka stream, each element in the stream is then consumed and
-added to the `resource` Kafka topic to be processed by the Policy service.
+> :warning:
```

Comment:
My vote would be keep the overview as the first section of the readme.  Put it in a section that has a header Overview.  This could also be a header that is common to each readme.  The warning I think should be part of the installation section of the readme and under a header Note